src/transforms/add_tags.rs

```diff
@@ -68,7 +68,8 @@ impl AddTags {
 impl FunctionTransform for AddTags {
     fn transform(&mut self, output: &mut Vec<Event>, mut event: Event) {
         if !self.tags.is_empty() {
-            let tags = &mut event.as_mut_metric().series.tags;
+            // TODO Replace this with a method on `struct Metric`
+            let tags = event.as_mut_metric().tags_mut();
```

Comment:
Meant to be handled in this PR or a follow-up?

Reply:
I wasn't sure. I could eliminate `fn tags_mut` completely by moving the code to `struct Metric`, but it would likely be a little less efficient due to repeated checks on tags being `None`.

Hmm, I see there are duplicated methods for removing tags, one of which I just added. Duh.

Reply:
Done in 72a1bc128a8b4c929183226882627f76aaabaa44

Reply:
Nice.